.changeset/public-files-throw.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/astro': minor
+---
+
+Introducing `<experimental_PricingTable/>`
```

Comment:
We need a better changeset here, explaining what it does etc.
Ideally we'd link to the launch blog post